background.js

```diff
@@ -58,15 +58,37 @@ chrome.contextMenus.create({
       contexts: ["browser_action"]
 });
 
+chrome.contextMenus.create({
+	  id: "AddSiteToFilterList",
+      title: "Block this page",
+      contexts: ["browser_action"]
+});
+
 chrome.contextMenus.onClicked.addListener(function contextMenuHandler(info, tab) {
 		switch(info.menuItemId) {
 			case "FilterListMenu":
 				chrome.tabs.create({ url: '/filterList.html'});
 				break;
+			case "AddSiteToFilterList":
+				chrome.tabs.query({active: true, currentWindow: true}, function(tabs) {
+					chrome.storage.sync.get('blockedSites', function (data){
+						if(tabs.length>1){
+							alert('Something gone wrong. Sorry.');
+							throw new Error('passed more than one page to be blocked', 'background.js', '76')
```

Comment:
Nie wiem czy możemy napisać konkretną linię, to się zaraz zmieni. Lepiej dać coś w stylu "Returned more than one active and current windows"